src/modules/endpoint/sagas/loadCategories.js

```diff
@@ -24,7 +25,10 @@ function * loadCategories (
   }
 
   console.debug('Fetching categories')
-  const categoriesPayload = yield call(() => createCategoriesEndpoint(baseUrl).request({ city, language }))
+  const appSettings = new AppSettings()
+  const apiUrlOverride = yield call(appSettings.loadApiUrlOverride)
+
+  const categoriesPayload = yield call(() => createCategoriesEndpoint(apiUrlOverride || baseUrl).request({ city, language }))
```

Comment:
Those two cases `apiUrlOverride` is null or not should be tested in `loadCategories.spec.js`.